pyoozie/client.py

```diff
@@ -561,6 +561,25 @@ def jobs_submit_coordinator(self, xml_path, configuration=None):
             return coord
         raise exceptions.OozieException.operation_failed('submit coordinator')
 
+    def jobs_update_coordinator(self, coordinator_id, xml_path, configuration=None):
+        user = self._user or 'oozie'
+        coord = self._fetch_coordinator_or_action(coordinator_id)
+        if coord.status.is_active():
+            conf = xml._coordinator_submission_xml(user, xml_path, configuration=configuration)
+            if self._verbose:
+                self.logger.info('Preparing to update coordinator %s:\n%s', xml_path, conf)
+            reply = self._put('job/{}?action=update'.format(coordinator_id), conf)
+
+            if not reply and 'update' not in reply:
```

Comment:
What does the reply look when the op fails? In particular, the case where an un-update-able field is changed may be worth handling with a different exception.